### PR TITLE
chore(serve): refactor code to be more concise

### PR DESCRIPTION
**Pull request by jamesgeorge007**
**What kind of change does this PR introduce?**
refactoring

**Did you add tests for your changes?**
Nope

**If relevant, did you update the documentation?**
N/A

**Summary**
Refactored the serve action handler to a more concise form.

**Does this PR introduce a breaking change?**
Nope

**Other information**
N/A

### CODE REVIEW

**Comment by jamesgeorge007** [comment]
cc @evenstensberg @ematipico

**Comment by ematipico** [comment]
The commit message should be `chore(serve)`
`feat()` <= this addresses a new feature, in this case :)

**Comment by jamesgeorge007** [comment]
@evenstensberg Thoughts :thinking:

File: packages/serve/index.ts [hunk 1]
```diff
@@ -22,30 +22,41 @@ import { processPromise } from "@webpack-cli/utils/resolve-packages";
...
+		dependency: ["add"],
+		devDependency: ["add", "-D"],
+		optionalDependency: ["add", "--optional"]
+	}
 };
 
 const spawnWithArg = (pm: string, cmd: string): SpawnSyncReturns<Buffer> => {
-	const pmConfig: ConfigType = pm === "npm" ? npmConfig : yarnConfig;
-	const options: string[] = [pmConfig.installCmd, "webpack-dev-server", pmConfig[cmd]];
+	/*
```
Comment (evenstensberg) [inline packages/serve/index.ts]: You can move this to a function doc if you see convention used a lil bit further up

**Comment by jamesgeorge007** [comment]
cc @evenstensberg

**Comment by jamesgeorge007** [comment]
cc @ematipico

**Comment by jamesgeorge007** [comment]
@ematipico Are we all set to merge this one :thinking:

**Comment by jamesgeorge007** [comment]
cc @evenstensberg

**Pull request merged**